exporter/collector/logs.go

```diff
@@ -102,14 +112,47 @@ func (l *LogsExporter) PushLogs(ctx context.Context, ld pdata.Logs) error {
 func (l logMapper) logToEntry(
 	log pdata.LogRecord,
 	mr *monitoredres.MonitoredResource,
+	instrumentationSource string,
+	instrumentationVersion string,
 ) (logging.Entry, error) {
 	entry := logging.Entry{
 		Resource: mr,
 	}
 
-	logBody := log.Body().BytesVal()
-	if len(logBody) > 0 {
-		entry.Payload = json.RawMessage(logBody)
+	entry.Labels = make(map[string]string)
+	if len(instrumentationSource) > 0 {
+		entry.Labels["instrumentation_source"] = instrumentationSource
```

Comment:
For metrics, we control whether or not these are added with a config option: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/collector_logging/exporter/collector/config.go#L75.  We might want to bring that option to the top-level of the config and use it for all types of telemetry (including here).  Maybe just add a TODO for now.